### DISCUSSION TITLE
Server Pro Console - Devices list, more devices per page...

### DISCUSSION THREAD

**Discussion by DMSnz** [rustdesk/rustdesk-server-pro#190] [Ideas]
Would it be possible to request the option to;
1. Have more than 100 devices per page (unlimited/all possibly)
2. Set cookie or whatever to save this preference as it resets every time you reload the page.

Cheers, we are sitting at close to 200 devices and this would be very helpful.

**Comment by rustdesk** [comment]
@21pages take care of this. This requirement makes sense to me also.

**Reply by rustdesk** [reply]
We can not do 1 for the time being. But please implement 2 first, (and make this available to all the other pages)
